xbmc/network/Network.cpp

```diff
@@ -466,6 +470,40 @@ bool CNetwork::PingHost(unsigned long ipaddr, unsigned short port, unsigned int
   return err_msg == 0;
 }
 
+bool CNetwork::PingHost(unsigned long remote_ip, unsigned int timeout_ms)
+{
+#ifdef TARGET_WINDOWS
+  return false;
```

Comment:
I like the idea of using Androids Network implementation, but we all try in the past to avoid ifderrery.
Moving PingHost from LinuxNetwork to Network makes it worse,

Pls. let us go and make it better, means something like a new file xbmc/network/android/Network.cpp/h.
The Darwin code is not fine as well, and will hopefully be moved to its own platform place next time someone touches it.